doc/user_manual/existing_interfaces.tex

```diff
@@ -2591,3 +2591,138 @@ \subsubsection{DataObjects}
+\begin{lstlisting}[language=bash]
+aprun -n 9 -d 32 /projects/sciteam/bahg/serpent/src/sss2 -omp 32 [inputFile]
+\end{lstlisting}
+where the \texttt{inputFile} is defined in the \xmlnode{Files} node as:
+\begin{lstlisting}[style=XML]
+<Files>
+    <Input name="originalInput" type="">serpent_input.serpent</Input>
+</Files>
+\end{lstlisting}
+
+\subsubsection{Files}
+The only input file needed is a complete SERPENT input file,
+which means that it should either be self sufficient, or includes
+the necessary files (e.g. geometry, material definition files).
+The input file extension will depend on the the extension definition
+in the \xmlnode{Models} node.
+
+\subsubsection{Samplers / Optimizers}
+In the \xmlnode{Samplers} block the user can define the variables
+to be sampled.
+\textbf{Example:} If the user wants to vary depletion time from
+10 to 100 days, the SERPENT input file \texttt{dep} definition is as such:
+\begin{lstlisting}
+dep daystep 
+$RAVEN-deptime$ 
+\end{lstlisting}
+
+Then in the RAVEN input file, the \xmlnode{Samplers} block would be:
+\begin{lstlisting}[style=XML]
+  <Samplers>
+    <Grid name="myGrid">
+      <variable name="deptime">
+        <distribution>timedist</distribution>
+        <!-- equally spaced steps with lower and upper bound -->
+        <grid construction="equal" steps="100" type="CDF">0.1 1</grid>
+      </variable>
+    </Grid>
+  </Samplers>
+\end{lstlisting}
+where the distributiong \texttt{timedist} is defined in the \xmlnode{Distributions}:
+\begin{listlisting}[style=XML]
+  <Distributions>
+    <!-- uniform distribution from 0.1 to 1 -->
+    <Uniform name="timedist">
+      <lowerBound>0.0</lowerBound>
+      <upperBound>100</upperBound>
+    </Uniform>
+</Distributions>
+\end{lstlisting}
+
+Then this run would be defined in the \xmlnode{Steps} block as \xmlnode{MultiRun}:
+\begin{lstlisting}[style=XML]
+<MultiRun name="runGrid">
+      <!-- runGrid runs serpent by the number of steps with sampled variable -->
+      <Input   class="Files"       type=""          >originalInput</Input>
+      <Model   class="Models"      type="Code"      >SERPENT</Model>
+      <Sampler class="Samplers"    type="Grid"      >myGrid</Sampler>
+      <Output  class="DataObjects" type="PointSet"  >outPointSet</Output>
+</MultiRun>
+\end{lstlisting}
+
+\subsubsection{Output Files Conversion}
+A single SERPENT run creates at least three output files,
+\texttt{[input]\_res.m, [input].out, [input]\_dep.m}.
+Since the interface is
+focused on acquiring the depleted composition, the \texttt{[input].bumat[n]}
+file is used as well. 
+
+The interface parses through \texttt{[input]\_res.m} file
+to obtain the Beginning Of Cycle and End Of Cycle Keffs, and the 
+\texttt{[input].bumat0} and \texttt{[input].bumat1} file to obtain the
+input / output compositions. In the output csv file, the input compositions
+are prepended with the letter `f', and the output compositions with the letter `d'
+(e.g. f92235, d94239), for separation. 
```

Comment:
Please add a snapshot of the SERPENT outputs here (very small or made up) and the resulting CSV structure (very small as well)